imagej/imagej-ops2/src/main/java/net/imagej/ops2/coloc/icq/LiICQ.java

```diff
@@ -53,11 +53,6 @@
  *            Type of the second image
  */
 @Plugin(type = Op.class, name = "coloc.icq")
-@Parameter(key = "image1")
```

Comment:
The commit message looks suspiciously like something I would write. Did I do this work, or did you? Or both together? If I was involved, I should've been included in the commit message with a `Co-authored-by` line at the end of the description. Not worth it to rewrite history just for that, but for future reference.

Reply:
Yes, if I remember correctly you wrote the initial script, which I then tweaked to produce the changes in this commit. I am sorry that your co-authorship did not persist 😦